src/bootstrap/bootstrap.py

```diff
@@ -463,10 +463,10 @@ def build_triple(self):
             cputype = 'i686'
         elif cputype in {'xscale', 'arm'}:
             cputype = 'arm'
-        elif cputype in {'armv6l', 'armv7l', 'armv8l'}:
+        elif cputype == 'armv6l':
             cputype = 'arm'
             ostype += 'eabihf'
-        elif cputype == 'armv7l':
+        elif cputype in {'armv7l', 'armv8l'}:
```

Comment:
`armv8l` is still running in 32-bit mode.  If you're running 64-bit, `uname -m` should say `aarch64`.

I just verified this on my Galaxy S7.  When I run `uname -m` via `adb shell`, I get `aarch64`.  When I run under JuiceSSH `localhost`, I get `armv8l`.  Last time I checked, Termux only installed a 32-bit environment, which might explain why you don't see `aarch64`.